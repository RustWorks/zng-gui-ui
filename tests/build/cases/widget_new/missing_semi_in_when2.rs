use zero_ui::prelude::*;

fn main() {
    let _scope = APP.minimal();
    let _ = Wgt! {
        margin = 0;
        cursor = CursorIcon::Default;
        when *#is_pressed {
            margin =
            cursor = CursorIcon::Pointer;
        }
    };
}
