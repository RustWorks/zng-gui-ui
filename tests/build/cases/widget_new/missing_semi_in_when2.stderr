error: expected `,`
  --> cases/widget_new/missing_semi_in_when2.rs:11:13
   |
11 |             cursor = CursorIcon::Hand;
   |             ^^^^^^

error[E0061]: this function takes 1 argument but 0 arguments were supplied
  --> cases/widget_new/missing_semi_in_when2.rs:10:13
   |
10 |             margin =
   |             ^^^^^^ an argument is missing
   |
note: associated function defined here
  --> $WORKSPACE/zero-ui/src/properties/layout.rs
   |
   | #[property(layout, default(0))]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = note: this error originates in the attribute macro `property` (in Nightly builds, run with -Z macro-backtrace for more info)
help: provide the argument
   |
10 |             margin(/* value */) =
   |                   +++++++++++++
